tests/queries/0_stateless/03322_materialized_view_ignore_errors_url.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS src;
+DROP TABLE IF EXISTS dst;
+DROP TABLE IF EXISTS mv;
+
+CREATE TABLE src (x int) ORDER BY ();
+
+CREATE TABLE dst (x int) AS url('http://127.0.0.1/', JSONEachRow, 'x int');
+
+CREATE MATERIALIZED VIEW mv TO dst AS SELECT * FROM src;
+
+INSERT INTO src SETTINGS materialized_views_ignore_errors = 0 VALUES (1); -- { serverError POCO_EXCEPTION }
+
+INSERT INTO src SETTINGS materialized_views_ignore_errors = 1 VALUES (2);
+
+SELECT * FROM src;
+
+DROP TABLE src;
+DROP TABLE dst;
+DROP TABLE mv;
+
+-- Ensure this still fails
+insert into function url('http://0.0.0.0/foo.tsv', 'TabSeparated', 'key Int') settings http_max_tries=1, materialized_views_ignore_errors=1 values (2); -- { serverError POCO_EXCEPTION }
```

Comment:
```suggestion
insert into function url('http://127.0.0.1/foo.tsv', 'TabSeparated', 'key Int') settings http_max_tries=1, materialized_views_ignore_errors=1 values (2); -- { serverError POCO_EXCEPTION }
```

I am a little bit worrying about differences between `0.0.0.0` and `127.0.0.1`, since they may get different network errors, though now they are the same.

Let's either use the same for both test cases, or even cover both addresses

Reply:
Ok, I'd say let's do 127.0.0.1.

There are many different potential network errors, and it would be nice to cover more of them, but it would take a long time to write reliable test cases. The original error that made me want to fix this was writing to a remote host that didn't have a proper SSL setup and getting `DB::Exception: SSL Exception: error:0A000086:SSL routines::certificate verify failed. (POCO_EXCEPTION)`.

But for example, what would happen if establishing the network connection and sending the data takes longer than the query timeout?

Reply:
>But for example, what would happen if establishing the network connection and sending the data takes longer than the query timeout?

It will NetException, that is inherited from Poco::Exception (P.S. there are two types of NetException - DB:: and Poco:: but both are inherited from Poco::Exception)